proto/ssz_proto_library.bzl

```diff
@@ -20,6 +20,7 @@ mainnet = {
     "sync_committee_bits.type": "github.com/prysmaticlabs/go-bitfield.Bitvector512",
     "sync_committee_aggregate_bytes.size": "16",
     "sync_committee_aggregate_bits.type": "github.com/prysmaticlabs/go-bitfield.Bitvector128",
+    "withdrawal.size": "16",
```

Comment:
Can this be linked to the minimal preset somehow? We define `MAX_WITHDRAWALS_PER_PAYLOAD` there, it'd be good if we could automatically change one if we change the other

Reply:
Out of my knowledge zone. I'm not sure how to do that. Looking at other variables here, I don't think we've done it before. cc @prestonvanloon for input

Reply:
I don't think we can do this with the existing tooling. However, we rarely change this file and all the values are contained in the same file so it may not be worth the trouble of deduplicating. I would approve as is

Reply:
The clear separation of minimal and mainnet in the same file makes this deduplication less of an issue as well

Reply:
Probably. Someone has to go figure out though